### PR TITLE
Fix bugs for scanner UI testing round 1

### PR DESCRIPTION
**Pull request by AllForNothing**
Fix UI bugs:
#9723 
#9721 
#9709 
#9708 
#9707 
#9706 
#9699
Signed-off-by: sshijun <sshijun@vmware.com>

### CODE REVIEW

File: src/portal/lib/src/tag/tag.component.scss [hunk 1]
```diff
@@ -255,4 +255,12 @@ clr-datagrid {
 .margin-top-m4{
   margin-top: -4px;
 }
-
+:host::ng-deep clr-dg-row {
+  .datagrid-select {
+    .clr-checkbox-wrapper {
+      input[type=checkbox] + label::before {
+        top: 0.3rem;
```
Comment (zhoumeina) [inline src/portal/lib/src/tag/tag.component.scss]: try margin:0;

File: src/portal/lib/src/vulnerability-scanning/result-grid.component.ts [hunk 2]
```diff
@@ -44,6 +51,29 @@ export class ResultGridComponent implements OnInit {
             this.loadResults(this.repositoryId, this.tagId);
         });
     }
+    getLevel(v: VulnerabilityItem) {
+        if (v) {
```
Comment (zhoumeina) [inline src/portal/lib/src/vulnerability-scanning/result-grid.component.ts]: define the return value in a const or enum.

File: src/portal/lib/src/tag/tag.component.scss (hunk 1 above)
Comment (AllForNothing) [reply src/portal/lib/src/tag/tag.component.scss]: Modified

File: src/portal/lib/src/vulnerability-scanning/result-grid.component.ts (hunk 2 above)
Comment (AllForNothing) [reply src/portal/lib/src/vulnerability-scanning/result-grid.component.ts]: Fixed

**Pull request merged**